src/lib/modules/Literature/LiteratureRelations.js

```diff
@@ -22,9 +22,7 @@ export default class LiteratureRelations extends Component {
 
   renderMultiparts = () => {
     const relations = _get(this.relations, 'multipart_monograph', []);
-    if (!relations.length) {
-      return null;
-    }
+    if (_isEmpty(relations)) return null;
```

Comment:
not sure I understand this change: if undefined, it will be set to `[]`. Did you found issues with all this?

Reply:
you are right on these ones I change them pro-actively to avoid comparison with `undefined` as you mentioned. I revert